README.md

```diff
@@ -1 +1,16 @@
-# Building-With_Active-Records
\ No newline at end of file
+# Micro-Reddit Project from Microverse
+
+This project puts your knowledge of active record, models, and associations to the tests. You will build an application similar to Reddit (called Micro-Reddit) where a user can create a post and add comments to it.
+
+In this project, you’ll build the data structures necessary to support link submissions and commenting. We won’t build a front end for it because we don’t need to… you can use the Rails console to play around with models without the overhead of making HTTP requests and involving controllers or views.
+
+## Installation
```

Comment:
- [ ] The readmefile can be improved, get some inspiration from the [microverse suggested template](https://github.com/microverseinc/readme-template)